.copier-answers.yml

```diff
@@ -15,7 +15,9 @@ odoo_test_flavor: Both
 odoo_version: 14.0
 org_name: Odoo Community Association (OCA)
 org_slug: OCA
-rebel_module_groups: []
+rebel_module_groups:
+- product_internal_reference_generator
```

Comment:
You only need to put one of them (yours), not both.

Reply:
updated